xml/selinux.xml

```diff
@@ -313,99 +291,45 @@ system_u:object_r:var_t var</screen>
    recommended to use a modular policy and not a monolithic policy. Modular
    policies are much easier to manage.
   </para>
- </sect1> -->
+ </sect1>
  
  <sect1 xml:id="sec-selinux-install">
-  <title>Installing &selnx; packages and modifying &grub;</title>
-
+  <title>Installing &selnx; packages</title>
   <para>
-   The easiest way to make sure that all &selnx; components are installed is
-   by using &yast;. The procedure described below shows what to do on an
-   installed &productname;:
+    If you are installing &selnx; with Zypper, search for all of the relevant
```

Comment:
captial Z with zypper?

Reply:
yes, as the product name, and not the command